src/borg/remote.py

```diff
@@ -900,6 +900,7 @@ def check(self, repair=False, save_space=False, max_duration=0):
 
     @api(since=parse_version('1.0.0'),
          compact={'since': parse_version('1.2.0a0'), 'previously': True, 'dontcare': True},
+         threshold={'since': parse_version('1.2.0a0'), 'previously': 0, 'dontcare': True},
```

Comment:
I guess "previously" should be 0.1 (the value resulting in the behaviour before introducing this parameter).

Reply:
I'm new to Python. I came quite far, but didn't yet find where the @api decorators are defined. Also didn't see why the test failed. If you point to where @api docs are/should be, I'll try to improve it.

Reply:
search for "def api".

Reply:
(but that is for a separate PR please)

Reply:
OK, tried to add doc in #4801